appengine/standard-cloud-tasks/app.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 202 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+STDOUT.sync = true
```

Comment:
Might be worth moving this inside the sample tags, along with a comment noting that this is a recommended practice to minimize delays in log transmission.

Reply:
done